board-viewer.html

```diff
@@ -1739,7 +1740,19 @@
     this.fire('item-hover', info);
   },
 
+  _isTouchDevice: function() {
```

Comment:
Does this work on touch enabled laptops? I can use either the screen or a mouse as input on mine (and frequently I use both at the same time... touch screen to tap items, mouse to navigate, scroll, etc.)

Instead of trying to ignore the events based on if it is a touchscreen, can we ignore the events based on if the system received a tap event first? When I was testing, I saw:

# via mouse
  mouseout (old item)
  mouseover (new item)
  mousetap (new item)

# via touch
  mousetap (new item)
  mouseover (new item)

It seemed the problems were being caused by:

a) No 'mouseout' ever being called
b) 'mouseover' being called after tap, combined with (a) left items highlighted in the board-viewer